client/src/components/Workflow/Invocations.vue

```diff
@@ -33,11 +34,27 @@
                         />
                     </b-card>
                 </template>
+                <template v-slot:cell(expand)="data">
+                    <b-link
+                        v-b-tooltip.hover.bottom
+                        title="Show Invocation Details"
+                        class="btn-sm fa fa-chevron-down"
+                        v-if="!data.detailsShowing"
+                        @click.stop="swapRowDetails(data)"
+                    />
+                    <b-link
+                        v-b-tooltip.hover.bottom
+                        title="Hide Invocation Details"
+                        class="btn-sm fa fa-chevron-up"
+                        v-if="data.detailsShowing"
+                        @click.stop="swapRowDetails(data)"
+                    />
+                </template>
                 <template v-slot:cell(workflow_id)="data">
                     <b-link
                         class="toggle-invocation-details"
                         v-b-tooltip
-                        title="Show Invocation details"
+                        :title="getWorkflowNameByInstanceId(data.item.workflow_id)"
```

Comment:
Is the the `toggle-invocation-details` class still right here ? And I guess we should have the same tooltip for the history name, which could also be too large

Reply:
Yes, I left the `toggle-invocation-details` class there and you can still expand the details if you click the name so the previous mechanism is still working for people that are used to it.
I agree the history name column should behave exactly the same and also overflow with ellipsis too instead of wrapping for consistency. Will do that :+1:

Reply:
Sounds good, but maybe you can move the `toggle-invocation-details` class to the new button ? I do want to add more context to the workflow and history cells anyway (filter by workflow / history for instace), but I can do that in dev

Reply:
Ok, I moved the class to the new button but still left the expand row details working on the workflow name, you can remove it on dev if you need it when you add the new stuff.